features/application/Titles.feature

```diff
@@ -18,13 +22,13 @@ Feature: application / titles
       | /onboarding/dataplanes-overview | Data plane overview |
       | /onboarding/completed           | Completed           |
 
-      | /zones/-create                                   | Create & connect Zone |
-      | /zones/zone-cps                                  | Zone Control Planes   |
-      | /zones/zone-cps/zone-cp-name/overview            | zone-cp-name          |
-      | /zones/zone-ingresses                            | Ingresses             |
-      | /zones/zone-ingresses/zone-ingress-name/overview | zone-ingress-name     |
-      | /zones/zone-egresses                             | Egresses              |
-      | /zones/zone-egresses/zone-egress-name/overview   | zone-egress-name      |
+      | /zones/-create                                                    | Create & connect Zone |
+      | /zones/zone-cps                                                   | Zone Control Planes   |
+      | /zones/zone-cps/zone-cp-name/overview                             | zone-cp-name          |
+      | /zones/zone-cps/zone-cp-name/ingresses                            | Ingresses             |
+      | /zones/zone-cps/zone-cp-name/ingresses/zone-ingress-name/overview | zone-ingress-name     |
+      | /zones/zone-cps/zone-cp-name/egresses                             | Egresses              |
+      | /zones/zone-cps/zone-cp-name/egresses/zone-egress-name/overview   | zone-egress-name      |
```

Comment:
I don't understand why we still have such a nesting.

Shouldn't it be:

```
/zones
/zones/-create
/zones/<zone-name>/overview
/zones/<zone-name>/config
/zones/<zone-name>/ingresses
/zones/<zone-name>/ingresses/<ingressName>/overview
/zones/<zone-name>/egresses
/zones/<zone-name>/egresses/<egressName>/overview
```

Reply:
Yeah sure we can do that I think, I was just following what was there already I suppose.

Ah, also when I did this work it was  before we added back the `-create` (DNS hyphen trick), without that we wouldn't be able to have a URL like `/zone/create` (due to maybe having a zone called `create`).

Lemme have a look now this has been rebased with the `-create` thing.

Reply:
Ah one other things I've spotted, I doubt its an issue, but thought I'd best mention incase people have opinions on it:

If I remove the `/zones/zone-cps` and use `/zones` instead.

1. When Zones are disabled you have: `/zones/egresses` (the egress listing)
2. When Zones are enabled, I can have a zone called `egresses` at `/zones/egresses` (the zone called `egresses` details page), and then `/zones/egresses/egresses` would be its egresses.

That would mean that 1 unique URL can point to 2 different 'documents' depending on whether you have zones enabled or not.

I don't think it has any other consequences, but would rather point it out incase

Lemme know, in the meantime I'm changing all this to remove the `zone-cps` segment as I'm guessing we are fine with this.

Reply:
K `/zone-cps/` has gone, we are now like this:

```
/zones
/zones/-create
/zones/<zone-name>/overview
/zones/<zone-name>/config
/zones/<zone-name>/ingresses
/zones/<zone-name>/ingresses/<ingressName>/overview
/zones/<zone-name>/egresses
/zones/<zone-name>/egresses/<egressName>/overview
```

will resolve this one.